fsspec/dircache.py

```diff
@@ -1,9 +1,13 @@
+import logging
 import time
 from collections.abc import MutableMapping
 from functools import lru_cache
+from pathlib import Path
 
+logger = logging.getLogger("fsspec")
```

Comment:
perhaps fsspec.dircache?

Reply:
The name is now taken from the file: `logging.getLogger(__name__)`